policyengine_us/parameters/gov/states/id/tax/income/credits/gc/aged_amount.yaml

```
@@ -0,0 +1,16 @@
+description: Idaho provides this additional amount under the grocery tax credit for aged filers.
+values:
+  2021-01-01: 20
```

per the law ` TAXABLE YEARS BEGINNING AFTER DECEMBER 31, 2007 `
```suggestion
  2008-01-01: 20
```